conf/application.conf

```diff
@@ -149,7 +149,7 @@ microservice {
 }
 
 fromAddress.domain = "tax.service.gov.uk"
-transactionEngine.fromAddress.domain = "transaction-engine.tax.service.gov.uk"
+transactionEngine.fromAddress.domain = "confirmation.tax.service.gov.uk"
```

Comment:
This will work fine, but you could also change the default in TransactionEngineFromAddress if you wanted to?